src/core/components/App.test.tsx

```diff
@@ -25,7 +25,11 @@ describe('App', (): void => {
         jest.spyOn(ApolloReactHooks, 'ApolloProvider').mockImplementation(
             ({ children }: ApolloProviderProps<unknown>): JSX.Element => apolloWrapper(mockQueryResponse)({ children }),
         );
+        // Apollo queries need timeout, despite mock...
+        // https://github.com/airbnb/enzyme/issues/2153
+        await act(async () => await new Promise(resolve => setTimeout(resolve, 10)));
```

Comment:
you should be able to use `wait` from react testing library here to make it `await wait();`

Reply:
if it requires a certain wait time you can pass as param of `wait`

Reply:
I didn't know about this lib. Cool stuff, nice catch.

Reply:
you can also just straight up mock out apollo, we have done this elsewhere to get around this and the errors it can spam to the logs.